<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background: rgb(0, 23, 71);
font-family: Microsoft Sans Serif;
}

QPushButton{
background-color: rgb(1, 179, 191);
border-radius: 20px;
font-size: 40px;
color: rgb(0,0,0);
}

QPushButton:hover{
background-color: rgb(77, 242, 243);
}


</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
qproperty-alignment: AlignCenter;
font-style: bold;
font-size: 35px;
color: white;
}</string>
      </property>
      <property name="text">
       <string>DEFINIR SEÑAL SEGÚN:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="MainTitleLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>71</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
qproperty-alignment: AlignCenter;
font-family: Microsoft Sans Serif;
font-size: 50px;
border-top: 3px solid white;
border-bottom: 3px solid white;
color: white;
}</string>
      </property>
      <property name="text">
       <string>SEÑALES NO CONVENCIONALES</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_7">
         <property name="minimumSize">
          <size>
           <width>371</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>371</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(1, 179, 191);
border: none;
border-radius: 20px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
qproperty-alignment: AlignCenter;
font-style: bold;
font-size: 25px;
color: black;
}</string>
            </property>
            <property name="text">
             <string>Número de Símbolos:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbNumSimb">
            <property name="minimumSize">
             <size>
              <width>61</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color:   rgb(255,255,255);
border: none;
border-radius: 0px;
}


QComboBox{
	background-color: rgb(204, 204, 255);
	background: white;
	color: black;
	border: 2px solid black;
	font-size: 15px;
	qproperty-alignment: AlignCenter;
	border-radius: 0px;
}</string>
            </property>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_48">
      <property name="minimumSize">
       <size>
        <width>143</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>143</width>
        <height>75</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border: none;
background-color: transparent
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_108">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="BackButton">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>111</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(255, 177, 64);
font-size: 20px;
text-align: right;
border-radius: 25px
}

QPushButton:hover{
background-color: rgb(255, 202, 128);
}
</string>
         </property>
         <property name="text">
          <string>Atrás   </string>
         </property>
        </widget>
       </item>
      </layout>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: transparent;
border: none
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>back arrow.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>BackButton</zorder>
      <zorder>label_8</zorder>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="SimbolosButton">
         <property name="minimumSize">
          <size>
           <width>291</width>
           <height>111</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>291</width>
           <height>111</height>
          </size>
         </property>
         <property name="text">
          <string>SÍMBOLOS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="BasesButton">
         <property name="minimumSize">
          <size>
           <width>291</width>
           <height>111</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>291</width>
           <height>111</height>
          </size>
         </property>
         <property name="text">
          <string>BASES</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
